### PR TITLE
Add New Company - Instructure

### PR DESCRIPTION
**Pull request by Christian-Oleson**
- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [ ] You are an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: company-profiles/instructure.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Remote status
+
+*How* you work is much more important than *where* you work. We offer work-from-home flexibility and flex time off ('cause we're all grown-ups here, and we trust our employees to kick some buns and get stuff done).
+
+## Region
+
+Europe
+North America
+Oceania (this includes Australia)
```
Comment (dougaitken) [inline company-profiles/instructure.md:19]: Readme says worldwide but this has three continents. Which is correct?

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/instructure.md (hunk 1 above)
Comment (Christian-Oleson) [reply company-profiles/instructure.md:19]: My apologies, it is those three continents, but I wasn't sure what constitutes worldwide (e.g. must it literally be at every region in the example file?

Comment (dougaitken) [reply company-profiles/instructure.md:19]: Worldwide would be stated by the company directly. The list tries to be as impartial but accurate as possible but essentially yes, I wouldn't accept worldwide if I know a region was excluded. I wouldn't want someone in India applying coming from this list but being disappointed when the company reject them.

**Review by dougaitken — APPROVED** [review]

File: company-profiles/instructure.md (hunk 1 above)
Comment (Christian-Oleson) [reply company-profiles/instructure.md:19]: Totally makes sense. Thanks for the clarification!

**Pull request merged**